tests/sys/vm/page_fault_signal.c

```diff
@@ -73,8 +73,15 @@ ATF_TC_BODY(page_fault_signal__segv_maperr_1, tc)
 	int sz;
 
 	sz = getpagesize();
+#ifdef __CHERI_PURE_CAPABILITY__
+	/* Need a RW mapping to get a RW capability. */
+	p = mmap(NULL, sz, PROT_READ | PROT_WRITE, MAP_ANON, -1, 0);
+	ATF_REQUIRE(p != MAP_FAILED);
+	ATF_REQUIRE(mprotect(p, sz, PROT_READ) == 0);
+#else
 	p = mmap(NULL, sz, PROT_READ, MAP_ANON, -1, 0);
```

Comment:
```suggestion
	p = mmap(NULL, sz, PROT_READ | PROT_MAX(PROT_READ | PROT_WRITE), MAP_ANON, -1, 0);
```
Should do the right thing without an ifdef.

Reply:
Nice

Reply:
Done.